docs/04-reference/wingsdk-spec.md

```diff
@@ -876,7 +876,7 @@ struct ApiRequest {
   /** The request's body. */
   body: Json?;
   /** The request's headers. */
-  headers: Map<str, str>;
+  headers: Map<str, str>?;
```

Comment:
Maybe?

```suggestion
  headers: Map<str, str> = {};
```

Reply:
Interesting idea, maybe add to the issue about structs?

Reply:
I believe I've added this to the optionals spec.